modules/understanding-upgrade-channels.adoc

```diff
@@ -7,89 +7,80 @@
 // * updating/updating-disconnected-cluster.adoc
 
 [id="understanding-upgrade-channels_{context}"]
-= Understanding {product-title} upgrade channels
-
-In {product-title} 4.1, Red Hat introduced the concept of upgrade channels for
-recommending the appropriate upgrade versions to your cluster. Upgrade channels
-separate upgrade strategies and also are used to control the cadence of updates.
-Channels are tied to a minor version of {product-title}. For instance,
-{product-title} {product-version} channels will never include an upgrade to a
-4.5 release. This ensures administrators make an explicit decision to upgrade to
-the next minor version of {product-title}. Channels only control updates and
-have no impact on the version of the cluster you install; the
-`openshift-install` binary for a given patch level of {product-title} always
-installs that patch level.
-
-{product-title} {product-version}, which includes the upgrade from the previous 4.3
-release, has three upgrade channels to choose from:
+= {product-title} Upgrade Channels and Releases
+
+In {product-title} 4.1, Red Hat introduced the concept of channels for
+recommending the appropriate release versions for cluster upgrade. By controlling
+the pace of upgrades, these upgrade channels allow users to choose an upgrade
+strategy. Upgrade channels are tied to a minor version of
+{product-title}. For instance, {product-title} {product-version}
+upgrade channels will never include an upgrade to a 4.5 release. This ensures
```

Comment:
The hard coding of 4.5 looks wrong here. This is will be wrong when this document gets published for 4.4 . Also we include previous version in channels e.g. https://github.com/openshift/cincinnati-graph-data/blob/master/channels/fast-4.3.yaml but not future minor versions (Y releases)

Reply:
This, as written, is for 4.4. So it's saying, for example, that stable-4.4 will never include an upgrade to 4.5. We checked with docs and, unfortunately, currently they could not come up with a way to implement variables like 'product-version-previous' or 'product-version-next'  so we're left with having to hard-code them.

Reply:
Sorry I gave an wrong example. I meant 4.6.  I am guessing the need for implementing `product-version-next` has not came yet. @kalexand-rh Whats your suggestion on this?

Reply:
We discussed in other PR, answer here: https://github.com/openshift/openshift-docs/pull/18944#discussion_r366816103